src/sso/pages/Loading.tsx

```diff
@@ -78,14 +78,17 @@ const Loading = () => {
       </div>
```

Comment:
up on L58, we want to at to storage the LoginRequest, which has the external website's
- origin (verified)
- callbackPath

then on L81 and wherever else relevant we want to redirect with something like
```ts
const response = {
  success: false
  reason: LoginFailReason.UserCancelled
}
const url = origin + callback + "?response=" + JSON.stringify(response)
window.location.href = url
```

then in the Demo website we need to catch the response in the URL
```ts
const params = new URLSearchParams(window.location.search);

const response = JSON.parse(params.get('response'));
if (response.success) {
  // Suceeded
  const requests = response.requests
  ...
} else {
  // check response.reason
  // Do something
}
```

Probably both above functions should be in the SDK